hdf5.spec

```diff
@@ -283,9 +298,13 @@ do
   mkdir $mpi
   pushd $mpi
 %if (0%{?suse_version} >= 1500)
```

Comment:
I wonder if doing this distro conditional here (and elsewhere) is the right way of doing this or if the names in the list `%{mpi_list}` should be conditionalized per distro.